src/web/testing/mocks.ts

```diff
@@ -1,23 +1,84 @@
-import { createQueryStringHash, encodeSymmetric } from 'atlassian-jwt';
+import {
+	createQueryStringHash,
+	encodeAsymmetric,
+	encodeSymmetric,
+} from 'atlassian-jwt';
+import { AsymmetricAlgorithm } from 'atlassian-jwt/dist/lib/jwt';
 import type { Method } from 'axios';
 
-import type {
-	ConnectInstallation,
-	ConnectInstallationCreateParams,
-} from '../../domain/entities';
+import { generateKeyPair } from 'node:crypto';
+import { promisify } from 'util';
 
 /**
- * Generates a JWT token that can be used to authorise inbound requests in
+ * Generates an asymmetric JWT token that can be used to authorise lifecycle event requests in
```

Comment:
/nit Might be worth mentioning that you also need to mock the public key endpoint, and/or linking to https://developer.atlassian.com/cloud/jira/platform/security-for-connect-apps/#validating-installation-lifecycle-requests

Reply:
Great suggestion!

✅ Done!